ocs_ci/templates/mcg/s3cli-sts.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  namespace: openshift-storage
+  name: s3cli
+spec:
+  selector:
+    matchLabels:
+      app: s3cli
+  replicas: 1
```

Comment:
Since this is StatefulSet with a single replica, does it mean that when a new pod is deployed (eg. because of the original pod was running on a broken node), it will create a new pod with a new name and "identity", or would it deploy pod with a new name?

Reply:
I believe that the new pod will have different identity features (the only thing I can think of is the UID, but still), but since it's a StatefulSet, all pods are bound to use the same base name, with a `-{n}` suffix, where `n` is the replica ID.
i.e. - in this case, we'll always have a pod named `s3cli-0`. 
If the replica amount was higher, it'd be `s3cli-1`, s3cli-2`, etc.